data-access-analysis-tools/adql-recipes.rst

```diff
@@ -0,0 +1,222 @@
+.. Review the README on instructions to contribute.
+.. Review the style guide to keep a consistent approach to the documentation.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Do not delete the expected section title, reference or related comments provided from the template.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. This is the label that can be used for cross referencing this file.
+.. Recommended title label format is "Directory Name"-"Title Name" -- Spaces should be replaced by hyphens.
+.. _Data-Access-Analysis-Tools-Adql-Recipes:
+.. Each section should include a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link and explicit title using the syntax :ref:`link text <label-name>`.
+.. A warning will alert you of identical labels during the linkcheck process.
+
+
+############
+ADQL Recipes
+############
+
+.. This section should provide a brief, top-level description of the page.
+
+ADQL is the `Astronomical Data Query Language <https://www.ivoa.net/documents/ADQL/>`_.
+The language is used by the `IVOA <https://ivoa.net>`_ to represent astronomy queries posted to Virtual Observatory (VO) services, such as the Rubin LSST Table Access Protocol (TAP) service.
+ADQL is based on the Structured Query Language (SQL).
+
+ADQL can be used in both the Notebook and Portal aspects:
+ - :ref:`Data-Access-Analysis-Tools-TAP-NB`
+ - how to :ref:`Portal-Intro-ADQL-Queries` in the Portal aspect
+
+Learn more about the `TAP-accessible DP0.2 catalogs <https://dp0-2.lsst.io/data-products-dp0-2/index.html#catalogs>`__.
+
+.. Important::
+    If a query takes longer than you expect, please submit a `GitHub Issue <https://github.com/rubin-dp0/Support>`__
+    or post in the "DP0 RSP Service Issues" category of the Rubin Community Forum.
+    Staff with familiarity with Qserv are happy to investigate and to help tweak queries for optimal execution.
```

Comment:
Unsure if this is necessary to define or link to more information, but the first time I heard Qserv I didn't know what it was. So this could be useful. I saw there's a github page, maybe this helps people? https://qserv.lsst.io/ Please feel free to ignore this comment.

Reply:
Oh. I see this is actually defined below. Maybe this red box is better positioned at the end of general advice? Again feel free to ignore suggestion.

Reply:
Good point -- Qserv doesn't need to be mentioned in the block. Reworded to simply _"Rubin staff are happy to investigate and to help tweak queries for optimal execution."_